aiidalab_widgets_base/computational_resources.py

```diff
@@ -1149,21 +1141,16 @@ class ComputerDropdownWidget(ipw.VBox):
     """Widget to select a configured computer.
 
     Attributes:
-        selected_computer(Unicode or Computer): Trait that points to the selected Computer instance.
-            It can be set either to an AiiDA Computer instance or to a computer label (will
-            automatically be replaced by the corresponding Computer instance). It is linked to the
+        selected_computer(Unicode of computer UUID): Trait that points to the selected Computer instance.
```

Comment:
Should we keep support for using a computer label instead of UUID?

Reply:
As the comment above, my initial plan is to only use UUID, since I think it is only used internally for the widget. The label will be shown from the key of `computer(Dict)` trait.
IMO, the value should be as static/identical as possible.

Reply:
I prefer to keep UUID only, tbh. See my reasoning above.